migrations/migrations/20180309234934-create-user.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+module.exports = {
+  up(queryInterface, Sequelize) {
+    return queryInterface.createTable('Users', {
```

Comment:
I dont know how you feel about it but, at least in postgres, tables cant have capital letters in their names so I prefer to use lowercase for the model names. It shouldnt really change anything since you are going to be doing everything through the orm but I dont like it that in the db if you use upper case you will get the name `public."Users"` instead of `public.users` . This is more my opinion though so you can choose what you prefer.

Reply:
completely agree! I didn't read the actual name of the table I assumed that it was going to create it following the conventions (lower case names)